### ISSUE TITLE
Can't find variable: document

### ISSUE THREAD

**Issue by nicolazj** [emotion-js/emotion#1429]
**Current behavior:**

I'm trying to use theme-ui in react native (expo 33), it throws:

ReferenceError: Can't find variable: document

the error is coming from @emotion/cache,  cache.browser.cjs.js

**Expected behavior:**

 the bundler in react native should use cache.cjs.js file.

I fixed the issue by adding below code to package.json file,  like the one in @emotion/core. but I am not sure if it's a proper way to fix.
```
 "react-native": {
    "./dist/cache.cjs.js": "./dist/cache.cjs.js",
    "./dist/cache.esm.js": "./dist/cache.esm.js"
  }
``` 

node modules infos:
    "@emotion/core": "^10.0.14",
    "@mdx-js/react": "^1.0.21",
    "expo": "^33.0.0",
    "react": "^16.8.3",
    "react-dom": "^16.8.6",
    "react-native": "https://github.com/expo/react-native/archive/sdk-33.0.0.tar.gz",
    "react-native-web": "^0.11.4",
    "theme-ui": "^0.2.6"

**Comment by Andarist** [comment]
That's the correct fix and there is a pending PR fixing this - https://github.com/emotion-js/emotion/pull/1412 . When using React Native please take [this](https://github.com/emotion-js/emotion/pull/1412#issuecomment-505246634) into consideration too.

**Comment by Andarist** [comment]
Closed by #1412
